.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contribution Guidelines
+
+## Reporting Issues
+
+[//]: # (Add proper project)
+[//]: # (Can be a github link)
+Please [open an issue](https://github.com/serokell/tezos-client/issues/new)
+if you find a bug or have a feature request.
+Before submitting a bug report or feature request, check to make sure it hasn't already been submitted
+
+The more detailed your report is, the faster it can be resolved.
+If you report a bug, please provide steps to reproduce this bug and revision of code in which this bug reproduces.
+
+
+## Code
+
+If you would like to contribute code to fix a bug, add a new feature, or
+otherwise improve our project, pull requests are most welcome.
+
+Our pull request template contains a [checklist](./pull_request_template.md#white_check_mark-checklist-for-your-pull-request) of acceptance criteria for your pull request.
+Please read it before you start contributing and make sure your contributions adhere to this checklist.
+
+
+## Legal
+
+We want to make sure that our projects come with correct licensing information
+and that this information is machine-readable, thus we are following the
+[REUSE Practices][reuse] – feel free to click the link and read about them,
+but, basically, it all boils down to the following:
+
+  * Add the following header at the very top (but below the shebang, if there
+    is one) of each source file in the repository (yes, each and every source
+    file – it is not as hard as it might sound):
+
+    ```haskell
+    {- SPDX-FileCopyrightText: 2019 Serokell <https://serokell.io>
```

Comment:
I am not sure, but I suspect the copyright holder is TQ. Let's wait until we get confirmation.

Reply:
Do all files should be with TQ as a holder?

Reply:
1. Files that are new, i. e. created during our work on `tezos-client` packaging.
2. Files that we copy-pasted from somewhere where TQ is the copyright holder.
3. Files that we copy-pasted from somewhere and that have `Unlicense` license (e. g. https://github.com/serokell/metatemplates/blob/master/templates/.github/CONTRIBUTING.md).

Reply:
In other words, files that shouldn't have TQ are the files that (we copy-pasted from somewhere) and (they have copyright holder other than TQ) and (their license doesn't allow to change copyright holder, i. e. we can assume that the license is not `Unlicense`).